x-pack/plugins/uptime/server/lib/saved_objects/synthetics_monitor.ts

```diff
@@ -18,27 +18,28 @@ export const syntheticsMonitor: SavedObjectsType = {
     dynamic: false,
     properties: {
       name: {
-        type: 'keyword',
+        type: 'text',
+        fielddata: true,
```

Comment:
without settings this, it would raise this error

`[2022-04-17T11:08:06.011+02:00][ERROR][http] ResponseError: all shards failed: search_phase_execution_exception: [illegal_argument_exception] Reason: Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [synthetics-monitor.name] in order to load field data by uninverting the inverted index. Note that this can use significant memory.`

Reply:
Why do we not need `fielddata` for other properties, do you know?

Reply:
@jportner any idea about this?

Reply:
I'm actually reviewing this now. I can see the error is related to the sorting feature in the monitor management list, and text fields not being optimized for sorting. This error is actually present when attempting to sort on the other fields we enable sorting for, urls and type, but does not apepar until you attempt to sort by those fields.

I am currently resolving this issue using multi fields and ensuring that sorting always uses the `.keyword` property. I will push directly to the branch.